### PR TITLE
Woocommerce2.0

### PR DESCRIPTION
**Pull request by pronob1010**
```diff
- WooCommerce integration. 
```

- [x] 1. Response method added and modified for data 
- [x] 2. Actions added (Update, delete, Quantity modification)

### CODE REVIEW

File: src/Api.php [hunk 1]
```diff
@@ -161,22 +198,174 @@ public function autonami_handler() {
...
+				if ( $item["product_id"] == (string) $product_id ) {
+					wc_update_order_item_meta( $item_id, '_qty', $quantity );
+					$order->calculate_totals();
+				}
+			}
+			$this->apiResponse->success( 200, [], 'Success' );
+		}
+	}
+
+	/**
```
Comment (atiq-ur) [inline src/Api.php:350]: Add type for params and do not need doc block, if not important.

**Review by atiq-ur — CHANGES_REQUESTED** [review]
Important:  Before merging into the develop branch, you must write the changelog in readme,txt, version.

1. Update version from thrivedesk.php file
2. Update changelog and version from readme.txt file
3. Update version at the package.json

File: src/Api.php [hunk 2]
```diff
@@ -16,8 +19,15 @@ final class Api {
 	private static $instance = null;
 
 	private $apiResponse;
-
 	private $plugin = null;
+	private $orderId = null;
```
Comment (Rijoanul-Shanto) [inline src/Api.php:23]: WordPress and Laravel coding guidelines are different. User lowercase (with an underscore) when declaring variable, action/filter, and function names.

Read this: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/

File: src/Plugins/WooCommerce.php [hunk 3]
```diff
@@ -220,19 +225,20 @@ public function order_status($order_id): array {
...
+	public function get_shipping_details( $order ): array {
+		$states = WC()->countries->get_states( $order->get_shipping_country() );
+		$state  = ! empty( $states[ $order->get_shipping_state() ] ) ? $states[ $order->get_shipping_state() ] : '';
 
 		$shipping_details = [];
 
-		array_push($shipping_details, [
-			'street'  => $order->get_shipping_address_1() ?? '',
+		array_push( $shipping_details, [
+			'street'  => $order->get_shipping_address_1() . ' ' . $order->get_shipping_address_2() ?? '',
```
Comment (alaminfirdows) [inline src/Plugins/WooCommerce.php:235]: `get_shipping_address_2() ?? ''`

or `$order->get_shipping_address_1() . ' ' . ($order->get_shipping_address_2() ?? ''),`

**Review by alaminfirdows — CHANGES_REQUESTED** [review]
Be aware about formating!

File: src/Api.php (hunk 2 above)
Comment (pronob1010) [reply src/Api.php:23]: done

File: src/Plugins/WooCommerce.php (hunk 3 above)
Comment (pronob1010) [reply src/Plugins/WooCommerce.php:235]: done

File: src/Api.php (hunk 1 above)
Comment (pronob1010) [reply src/Api.php:350]: done

**Pull request merged**